### ISSUE TITLE
Support Chrome Version History API

### ISSUE THREAD

**Issue by jay0lee** [jay0lee/GAM#1344]
The discovery doc doesn't work (static doc attached) but the endpoints themselves require no OAuth or key to use.

Beyond reporting on current version and current rollout status (fraction is the % of devices that will get the update as Google scales the rollout to more users over time so fraction==0.01 means 1% of users globally will get the update automatically). Anyway, beyond that it'd be good to pull this information in for things like browser version pinning Chrome policy so admins can say things like "pin users to the current stable version minus 1" or "pin to the latest release pushing at 100%".

https://developer.chrome.com/docs/versionhistory/guide/

[versionhistory-v1.json.txt](https://github.com/jay0lee/GAM/files/6205918/versionhistory-v1.json.txt)

**Comment by jay0lee** [comment]
@taers232c

**Comment by taers232c** [comment]
Not ready for prime time. I can't enable the API and the discovery document does't reference any scopes.

**Comment by jay0lee** [comment]
Agreed it needs work but you should be able to just use build_from_document and an HTTP object with no authentication.

**Comment by taers232c** [comment]
Working on this; is it always going to be unauthenticated?

```
gam print chromehistory platform [todrive]
gam print chromehistory channel [todrive]
        [platform <PlatformType>]
gam print chromehistory version [todrive]
        [platform <Platform>] [channel <ChannelType>]
        [filter <String>]
        [orderby channel|name|platform|version]
gam print chromehistory release [todrive]
        [platform <Platform>] [channel <ChannelType>] [version <String>]
        [filter <String>]
        [orderby channel|endtime|fraction||name|platform|starttime|version]
```

**Comment by jay0lee** [comment]
Not sure, discussing with team but change won't be immediate.

**Comment by jay0lee** [comment]
Ross implemented this in #1354
